bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/AbstractTreeViewer.java

```diff
@@ -2455,6 +2518,59 @@ public void setAutoExpandLevel(int level) {
 		expandToLevel = level;
 	}
 
+	/**
+	 * Sets the level of auto expand on paths of consecutive single children.
+	 * <code> NO_EXPAND </code> means that such paths are not automatically
+	 * expanded.
```

Comment:
I'd like to have a more precise description of the meaning of the given number of levels. E.g., does `level=1` mean that one _additional_ level is expanded in case an item with one child is expanded or that the the one level that is expanded anyway is expanded? Currently it seems to be latter one, i.e., `level=0` and `level=1` will have the same behavior. That behavior is fine for me, but should be precisely documented.